src/Packages/UnityWebBrowser/Runtime/Core/RawImageUwbClientInputHandler.cs

```diff
@@ -89,13 +98,18 @@ private IEnumerator KeyboardAndMouseHandler()
 
             while (Application.isPlaying)
             {
-                if(!browserClient.IsConnected)
+                yield return 0;
+
+                if (!browserClient.IsConnected)
                     continue;
-                
+
+                if(disableAllControls)
+                    continue;
+
                 if (GetMousePosition(out Vector2 pos))
                 {
                     //Mouse position
-                    if (lastSuccessfulMousePositionSent != pos)
+                    if (!disableMouse && lastSuccessfulMousePositionSent != pos)
                     {
                         browserClient.SendMouseMove(pos);
                         lastSuccessfulMousePositionSent = pos;
```

Comment:
I think it would be better to just have one if statement check for `disableMouse` with all the related mouse logic inside of it. Make so it is like:

```csharp
if(!disableMouse)
{
      //Mouse position
      if (lastSuccessfulMousePositionSent != pos)
      {
            browserClient.SendMouseMove(pos);
            lastSuccessfulMousePositionSent = pos;
      }

      //Mouse scroll
      ...
}
```

(I would highlight the entire area where the if statement should cover, but GitHub won't let me, sorry, code example should show what I want)